RELEASE-NOTES.md

```diff
@@ -134,21 +137,16 @@ If the file is valid YAML but does not contain a valid configuration, it prints
 
 #### Start node
 
-It is recommended that at least 1024 files can be opened in the session where you start the node: this is needed not only for files on disk (e.g. persisted chain) but also for network connections with peers.
-You can check your max number of open files:
-* On Ubuntu, by running `ulimit -n`;
-* On macOS, by running `ulimit -n`.
-
-If your max number of open files is lower than 1024, you are recommended to raise such configuration:
-* On Ubuntu, `ulimit -n 1024` shall be sufficient;
-* On macOS, `ulimit -n 1024` shall be sufficient; if not sufficient on your OS, please report also the output of `sysctl -a | grep files`.
+It is recommended that the node has at least 4 GB of memory available.
```

Comment:
I would keep a note regarding ulimit. Say that proper configuration of ulimit is esential to manage connections and that users should make sure they configure context of the epoch process